homeassistant/components/system_bridge/__init__.py

```diff
@@ -129,107 +109,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
 
-    if hass.services.has_service(DOMAIN, SERVICE_OPEN_URL):
-        return True
-
-    def valid_device(device: str):
-        """Check device is valid."""
-        device_registry = dr.async_get(hass)
-        device_entry = device_registry.async_get(device)
-        if device_entry is not None:
-            try:
-                return next(
-                    entry.entry_id
-                    for entry in hass.config_entries.async_entries(DOMAIN)
-                    if entry.entry_id in device_entry.config_entries
-                )
-            except StopIteration as exception:
-                raise vol.Invalid from exception
-        raise vol.Invalid(f"Device {device} does not exist")
-
-    async def handle_open_path(call: ServiceCall) -> None:
-        """Handle the open path service call."""
-        _LOGGER.info("Open: %s", call.data)
-        coordinator: SystemBridgeDataUpdateCoordinator = hass.data[DOMAIN][
```

Comment:
Refactoring & new features should not be put into the same PR. Please split it out.